stable/spring-cloud-data-flow/README.md

```diff
@@ -101,6 +127,19 @@ The following tables list the configurable parameters and their default values.
 
 ### MySQL Configuration
 
-| Parameter                  | Description           | Default                   |
-| -------------------------- | --------------------- | ------------------------- |
-| mysql.mysqlDatabase        | MySQL database name   | dataflow
+| Parameter                  | Description                  | Default                   |
+| -------------------------- | ---------------------------- | ------------------------- |
+| mysql.enabled              | Enable deployment of MySQL   | true
+| mysql.mysqlDatabase        | MySQL database name          | dataflow
+
+### External Database Configuration (mysql.enabled=false)
```

Comment:
this still needs to be changed.. it should be `### External Database Configuration`

up higher in the readme we already know this parameter needs to be set and if out of the box support for another DB comes along, this is yet another place to update. please remove, thanks.